ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -1140,6 +1140,15 @@ function seriesRenderCanvas(this: Series): void {
         minI: (number|undefined),
         maxI: (number|undefined);
 
+    // Remove k-d-tree mock points after zoom (#20330)
+    if (this.points) {
+        this.points.length = 0;
+    }
+
+    if (this.boosted && chart.hoverPoint?.series === this) {
+        chart.hoverPoint = null as any;
```

Comment:
Just a very nitpicky thing (and not even 100% if this is better 🤣) , but you can avoid the `any` cast by doing: `chart.hoverPoint = void 0;`

Reply:
thanks!